Gemfile

```
@@ -4,14 +4,14 @@ source "https://rubygems.org"
 
 ruby RUBY_VERSION
 
-DECIDIM_VERSION = { github: "decidim/decidim", ref: "release/0.26-stable" }.freeze
+DECIDIM_VERSION = { github: "decidim/decidim", tag: "v0.27.5" }.freeze
```

```suggestion
DECIDIM_VERSION = "0.27.5"
```
Let's just use this no?